collections/_article/godot-at-gdc-2024.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Godot Does Conferences - Meet us at GDC!"
+excerpt: "We are excited to announce that the Godot Engine will attend the 2024 Game Developers Conference (March 18-22)."
+categories: ["events"]
+author: Godot Foundation
+image: /storage/blog/gdc/2024/gdc-thumbnail.webp
+date: 2024-02-13 12:00:00
+---
+
+Since we had a great & productive time last year, and 2.000 pins dispersed around the world to prove it, we are attending the [Game Developer Conference](https://gdconf.com/) (GDC) again, which is taking place March 18-22 in San Francisco, California. 
```

Comment:
Sorry Europe. (Alternatively, just "2000" and there's no ambiguity.)
```suggestion
Since we had a great & productive time last year, and 2,000 pins dispersed around the world to prove it, we are attending the [Game Developer Conference](https://gdconf.com/) (GDC) again, which is taking place March 18-22 in San Francisco, California. 
```